edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/IoTHubMetricsUpload.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
```

Comment:
Same comment as above. I would expect this to have the namespace `Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Uploader`. The directory for this would be `edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/uploader/IoTHubMetricsUpload.cs`

Reply:
Same for `Serializer` above.